codespell_lib/data/dictionary.txt

```diff
@@ -8128,7 +8129,7 @@ correnspondents->correspondents
 corrensponding->corresponding
 corrensponds->corresponds
 corrent->correct, current,
-corrently->correctly
+corrently->correctly, currently,
```

Comment:
I think swap these round, your suggestion seems far more likely!

Reply:
I initially thought this as well.. but then I looked at line 8130 (above) and decided to copy the order based on that.

Reply:
Yeah actually grep.app seems to agree. Perhaps correct(ly) is just used more commonly.